test/run_envoy_bazel_coverage.sh

```diff
@@ -61,7 +61,7 @@ BAZEL_BUILD_OPTIONS="${BAZEL_BUILD_OPTIONS} -c dbg --copt=-DNDEBUG"
 "${BAZEL_COVERAGE}" test //test/coverage:coverage_tests ${BAZEL_BUILD_OPTIONS} \
   --cache_test_results=no --cxxopt="--coverage" --cxxopt="-DENVOY_CONFIG_COVERAGE=1" \
   --linkopt="--coverage" --define ENVOY_CONFIG_COVERAGE=1 --test_output=streamed \
-  --strategy=Genrule=standalone --spawn_strategy=standalone --test_timeout=4000 \
+  --strategy=Genrule=standalone --spawn_strategy=standalone --test_timeout=5000 \
```

Comment:
Per slack discussion is this really needed? Can you check the typical time of a passing test to see if we are really up against the limit or if there is a hang?

Reply:
don't know; I remember bringing this up on slack and did not notice a better answer. But I guess if other PRs are passing coverage at 4k seconds I can't see how the test I added would make it slower.

So I'll try reverting this and seeing if passes CI now.

Reply:
CI passes. Not sure what was going wrong before.